cmd/wksctl/profile/constants/constants.go

```diff
@@ -1,6 +1,7 @@
 package constants
 
 const (
-	AppDevAlias   = "app-dev"
-	AppDevRepoURL = "git@github.com:weaveworks/eks-quickstart-app-dev"
+	AppDevAlias          = "app-dev"
+	AppDevRepoURL        = "git@github.com:weaveworks/eks-quickstart-app-dev"
+	WKSctlIgnoreFilename = ".wksctlignore"
```

Comment:
It feels to me that `wksctl` in `wksctlignore` refers to our tool in an overly specific way. It is not impossible that we have another tool (other than `wksctl`) in the future that deals with profiles, for example.

Unless we're already committed to calling this `.wksctlignore`, I suggest renaming to `.profileignore`.

Reply:
@mflendrich basically yes, it is very specific to `wksctl`. Let me explain by referring to this example profile:
https://github.com/chanwit/eks-quickstart-app-dev

In the above profile, there are two ignore files `.eksctlignore` and `.wksctlignore`.
Each of them contains a set of file needs to be removed by each tool.

This is by design to make a single profile repo works on both `eksctl` and `wksctl`.
Thinking of this as a situation that we have `.circleci`, `.travis.yaml` and `.drone.yaml` files on a git repo to allow each CI to independently process our repo (in its own way).